integration/spark/src/main/scala/org/apache/carbondata/events/CleanFilesEvents.scala

```diff
@@ -26,5 +26,6 @@ case class CleanFilesPreEvent(carbonTable: CarbonTable, sparkSession: SparkSessi
 case class CleanFilesPostEvent(
     carbonTable: CarbonTable,
     sparkSession: SparkSession,
-    options: Map[String, String])
+    options: Map[String, String],
+    dryRun: Boolean)
```

Comment:
why not sending this in options itself ?

Reply:
changed